source/common/router/rds_impl.h

```diff
@@ -130,7 +130,7 @@ class RdsRouteConfigProviderImpl
   Upstream::ClusterManager& cm_;
   std::unique_ptr<Envoy::Config::Subscription<envoy::api::v2::RouteConfiguration>> subscription_;
   ThreadLocal::SlotPtr tls_;
-  std::string cluster_name_;
+  std::string cluster_config_;
```

Comment:
@junr03 i have changed this variable name and method name to mean what we are doing here. let me know if you want to use any better names